2-mandatory/6-journey-planner.js

```diff
@@ -123,6 +130,14 @@ function getLocationName() {}
 */
 function journeyPlanner(locations, transportMode) {
```

Comment:
Can you rewrite this function using the functions you implemented just a few lines above?

Reply:
I am not sure which function you are talking about it, sorry!